documentation/IDTA-01002-3/modules/ROOT/pages/Annex/IDTA-01002_SerializationModifierExamples.adoc

```diff
@@ -118,52 +119,15 @@ If applied to the Submodel:
         "value": "0173-1#01-AFZ615#016"
```

Comment:
```suggestion
        "value": "0173-1#01-AHF578#001"
```

Reply:
I did not check the older IRDI, but this one is for HandoverDocumentation, see https://github.com/admin-shell-io/submodel-templates/blob/main/published/Handover%20Documentation/1/2/IDTA%2002004-1-2_Submodel_Handover%20Documentation.pdf

Reply:
idShort should be changed to "HandoverDocumentation"